doc/classes/@GDScript.xml

```diff
@@ -972,6 +972,24 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="smoothstep">
+			<return type="float">
+			</return>
+			<argument index="0" name="from" type="float">
+			</argument>
+			<argument index="1" name="to" type="float">
+			</argument>
+			<argument index="2" name="weight" type="float">
+			</argument>
+			<description>
+				Returns a number smoothly interpolated between the [code]from[/code] and [code]to[/code], based on the [code]weight[/code]. Similar to [method lerp], but interpolates faster at the beginning and slower at the end.
```

Comment:
@Chaosus, actually this implementation of **smoothstep** is similar to [inverse_lerp](https://docs.godotengine.org/en/3.2/classes/class_@gdscript.html#class-gdscript-method-inverse-lerp) and not to [lerp](https://docs.godotengine.org/en/3.2/classes/class_@gdscript.html#class-gdscript-method-lerp) so this description is misleading. Also another difference (besides interpolating faster at the beginning and slower at the end) is that **smoothstep** clamps result to range <0, 1>. So this description probably needs an update.
_(no idea if that's proper place for this note)_